solr/solrj/src/java/org/apache/solr/client/solrj/io/stream/FacetStream.java

```diff
@@ -20,15 +20,19 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
+import java.util.HashMap;
```

Comment:
This source file did not adhere to the community code format so I reformatted it. It's bad practice in general, but since my PR was getting dinged for format issues, I chose to fix globally in this file vs. piecemeal.